### PR TITLE
Fix the reference to JSPlatform in the readme.

### PR DESCRIPTION
**Pull request by mariussoutier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ import org.portablescala.sbtplatformdeps.PlatformDepsPlugin.autoImport._
 import scala.scalanative.sbtplugin.ScalaNativePlugin.autoImport._
 import sbtcrossproject.CrossPlugin.autoImport._
 import scalajscrossproject.ScalaJSCrossPlugin.autoImport.{toScalaJSGroupID => _, _}
-import scalajscrossproject.JSPlatform
+import org.scalajs.sbtplugin.JSPlatform
```
Comment (sjrd) [inline README.md]: Actually, this should be `scalajscrossproject.ScalaJSCrossPlugin.autoImport.JSPlatform`.

The fact that it is still accessible as `org.scalajs.sbtplugin.JSPlatform` is a mistake. That one should be deprecated.

Comment (sjrd) [reply README.md]: Filed as #83.

**Comment by sjrd** [comment]
The CI is asking you to [sign the CLA](https://www.lightbend.com/contribute/cla/scala) ;)

**Comment by mariussoutier** [comment]
Hmm, I thought I already did that a long time ago. I contributed some smaller PRs to Play. Well, I resubmitted.

**Pull request merged**